### PR TITLE
feat(lsd-relayer): relayer ready to submit proof and storage to picasso/composable

### PR DESCRIPTION
**Pull request by RustNinja**
Required for merge:
- [ ] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](../terraform/github.com/branches.tf) 

Makes review faster:
- [ ] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [ ] I marked PR by `misc` label if it should not be in release notes
- [ ] Linked Zenhub/Github/Slack/etc reference if one exists
- [ ] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [ ] Added reviewer into `Reviewers`
- [ ] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [ ] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [ ] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

File: code/parachain/frame/liquid-staking/relayer/src/main.rs [hunk 1]
```diff
@@ -49,33 +99,84 @@ async fn main() {
...
 		.unwrap();
 
 	while true {
-		for i in &tuple {
-			let keys = vec![i.1.as_ref()];
+		for i in &sub_accounts {
+			let keys = vec![i.storage_key.as_ref()];
+
+			let api = OnlineClient::<subxt::SubstrateConfig>::from_url(para_ws_url).await.unwrap();
+			let block_hash_para = api.rpc().block_hash(None).await.unwrap().unwrap();
```
Comment (dzmitry-lahoda) [inline code/parachain/frame/liquid-staking/relayer/src/main.rs:159]: please rename api to para_api

File: code/parachain/frame/liquid-staking/relayer/src/main.rs [hunk 2]
```diff
@@ -88,21 +189,20 @@ async fn main() {
...
 			let Some(ledger) = relay_client.storage().at(block_hash).fetch(&staking).await.unwrap()
 			else {
-				println!("ledger not found");
+				println!("ledger: {} not found", i.address);
 				continue;
 			};
 
-			let sl = hyperspace_core::substrate::composable::relaychain::api::runtime_types::pallet_staking::StakingLedger::try_from(ledger).unwrap();
-			println!("sl: {:?}", sl);
+			let sl =
```
Comment (dzmitry-lahoda) [inline code/parachain/frame/liquid-staking/relayer/src/main.rs:202]: please rename sl

File: code/parachain/frame/liquid-staking/relayer/src/main.rs [hunk 3]
```diff
@@ -118,22 +218,18 @@ async fn main() {
 			}
 
 			let xxx = StakingLedger::<AccountId32, u128> {
```
Comment (dzmitry-lahoda) [inline code/parachain/frame/liquid-staking/relayer/src/main.rs:220]: xxx?ay be just x? or X?

**Comment by dzmitry-lahoda** [comment]
if possible.please split main into 2 files, once is just config, and other core logic/flow. so can read just core logic, without config noise

File: code/parachain/frame/liquid-staking/relayer/src/main.rs (hunk 1 above)
Comment (RustNinja) [reply code/parachain/frame/liquid-staking/relayer/src/main.rs]: Done.

**Comment by RustNinja** [comment]
> if possible.please split main into 2 files, once is just config, and other core logic/flow. so can read just core logic, without config noise

Done

**Review by dzmitry-lahoda — APPROVED** [review]

**Pull request closed without merging**